src/ed25519.rs

```diff
@@ -8,38 +8,38 @@
 // Authors:
 // - Isis Agora Lovecruft <isis@patternsinthevoid.net>
 
-//! A Rust implementation of ed25519 EdDSA key generation, signing, and
+//! A Rust implementation of ed25519 `EdDSA` key generation, signing, and
```

Comment:
*sigh*

"EdDSA" is neither a mathematical object nor a unit of code. Clippy is mistaken here.